app/lib/features/chat/widgets/bubble_builder.dart

```diff
@@ -206,7 +205,7 @@ class _ChatBubble extends ConsumerWidget {
     return Bubble(
       color: isAuthor
           ? Theme.of(context).colorScheme.surface
-          : Theme.of(context).colorScheme.neutral2,
+          : Theme.of(context).colorScheme.surface,
```

Comment:
`? Surface : surface` ... Sometimes colors has a function. Here is had one. Either you remove the entire check or use different colors... But if I recall correctly _we want_ differently colored bubbles for my messages and other people's messages so ... I guess it's time to define a different color...